pyuvdata/uvbeam.py

```diff
@@ -19,13 +19,15 @@ class UVBeam(UVBase):
     """
 
     coordinate_system_dict = {
-        # uniformly gridded az/za coordinate system, az runs from East to North in radians
-        'az_za': ['az', 'za'],
-        # sine projection at zenith. y points North, x point East
-        'sin_zenith': ['sin_x', 'sin_y'],
-        # HEALPix map with zenith at north pole and
-        # az, za coordinate axes (for efield) where az runs from East to North.
-        'healpix': ['hpx_inds']}
+        'az_za': {'axes': ['az', 'za'],
+                  'description': 'uniformly gridded az/za coordinate system, '
```

Comment:
Please spell out "azimuth" and "zenith angle" in the description.

Reply:
done